### DISCUSSION TITLE
RAZZLE_APP_PATH

### DISCUSSION THREAD

**Discussion by batusai513** [jaredpalmer/razzle#1595] [Q&A] [answered]
@fivethreeo Hi 👋🏼  I asked for this change a while back, the idea was to use razzle as a package builder for other packages in a monorepo setup, today I'm upgrading to razzle v4 and found out that RAZZLE_APP_PATH is not working for my case because this commit is making the RAZZLE_APP_PATH be inside the current working directory (https://github.com/jaredpalmer/razzle/commit/8056ffed3ed6b4a86aeb38a75bd76c0be0f02e02) which breaks the initial idea May I asked why this change?

**Comment by fivethreeo** [comment, marked as answer]
Was to allow relative paths, but  I am open to a change that tests if the path is absolute.

**Reply by batusai513** [reply]
I can make a PR with that change that will make our life easier and allow us to update to v4. Thanks for your fast response 🙇🏼
